tbtaf/cx_Oracle_test.py

```diff
@@ -0,0 +1,12 @@
+import cx_Oracle
+import os
+import pandas as pd
+
+os.environ['TNS_ADMIN'] = '/Users/david.torres/Documents/MCAP/system-desing/TBTAF/tbtaf/adb_virt_env'
```

Comment:
Please remove any hard-coded credentials and implement logic to fetch it from OS environment variables as designed.

Reply:
I pushed this file by mistake, sorry

Removed